crates/ruma/Cargo.toml

```diff
@@ -103,12 +103,14 @@ unstable-exhaustive-types = [
     "ruma-federation-api/unstable-exhaustive-types",
     "ruma-identity-service-api/unstable-exhaustive-types",
     "ruma-push-gateway-api/unstable-exhaustive-types",
+    "ruma-state-res/unstable-exhaustive-types"
 ]
 unstable-pre-spec = [
     "ruma-common/unstable-pre-spec",
     "ruma-client-api/unstable-pre-spec",
     "ruma-events/unstable-pre-spec",
     "ruma-federation-api/unstable-pre-spec",
+    "ruma-state-res/unstable-pre-spec"
```

Comment:
This is only used in tests so isn't needed right?

Reply:
I guess if we have the flag we might as well enable it. Maybe we'll actually make use of it outside of tests in the future.